database.go

```diff
@@ -157,20 +157,31 @@ func MigrateForwardTo(name string, db *sql.DB, allMigrations []Migration, confir
 	}
 	// run migrations
 	for _, mig := range toRun {
-		err = runMigrationSQL(db, mig.Name, mig.ForwardSQL)
+		err = runMigrationSQL(db, mig.Name, mig.ForwardSQL, mig.SeparateForwardStatements)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func runMigrationSQL(db *sql.DB, name, sqlToRun string) error {
+func runMigrationSQL(db *sql.DB, name, sqlToRun string, seperate bool) error {
 	fmt.Printf("Running %s... ", name)
-	_, err := db.Exec(sqlToRun)
-	if err != nil {
-		fmt.Println("Failure :(")
-		return fmt.Errorf("error running migration: %v", err)
+	if !seperate {
+		_, err := db.Exec(sqlToRun)
+		if err != nil {
+			fmt.Println("Failure :(")
+			return fmt.Errorf("error running migration: %v", err)
+		}
+	} else {
+		statementArr := strings.Split(sqlToRun, ";")
```

Comment:
I don't think it's safe to split this way.  If there were commented lines in forward.sql that had semicolons in them, they could end up getting executed.  Example:

```sql
-- this is a comment; DROP TABLE users;
```